### ISSUE TITLE
[Rogue] Killing Spree range is incorrect on bosses

### ISSUE THREAD

**Issue by mercred** [dalaranwow/dalaran-wow#5740]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Killing Spree places a rogue max melee range on bosses
**Expected behaviour**: Tell us what should happen instead
Killing Spree should place a rogue to the middle of the boss red circle with slight offset to the back.
**Steps to reproduce the problem**:

1. Cast KS on a boss

**Include proofs for this behaviour**
Killing Spree range on dala-wow
![Killing Spree rogue position](https://user-images.githubusercontent.com/15626206/59502493-2ea43080-8e9e-11e9-8455-40b24c689f79.png)
Killing Spree range on retail:
![image](https://user-images.githubusercontent.com/15626206/59502610-79be4380-8e9e-11e9-86a8-10c27fe155c4.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
